baseplate/lib/live_data/zookeeper.py

```diff
@@ -2,12 +2,47 @@
 from typing import Optional
 
 from kazoo.client import KazooClient
+from kazoo.client import KazooState
 from kazoo.handlers.gevent import SequentialGeventHandler
+from prometheus_client import Counter
 
 from baseplate.lib import config
 from baseplate.lib.secrets import SecretsStore
 from baseplate.server.monkey import gevent_is_patched
 
+SESSION_LOST_TOTAL = Counter(
```

Comment:
Added spec PR https://github.snooguts.net/reddit/baseplate.spec/pull/73

Reply:
Based on discussion on the [PR](https://github.snooguts.net/reddit/baseplate.spec/pull/73), perhaps we keep this to a sidecar specific metric now (since there are no other ZK clients in baseplate). If/when we have a Zookeeper use-case and baseplate client we can spec-ify and standardize the client.